ncm-ccm/src/main/pan/components/ccm/schema.pan

```diff
@@ -7,6 +7,7 @@ declaration template components/ccm/schema;
 
 include 'quattor/types/component';
 include 'pan/types';
+include 'components/accounts/functions';
```

Comment:
if the `group_readable` is configured and does not exist during `ccm-fetch` or `ccm-initialise` (the only parts of CCM that currently write; i.e. not this component), the group bits are not set by CCM (so `root.root` and possible world_readable).
only the next time around, new configuration settings will start to work. i have no clue how to make this better atm.
`ccm-fetch` will simply log error until the group is available, nothing more. (in particular, at present, `ncm-ccm` can't fix the group issue. even if it would log an error when the group does not exist (which it doesn't now); it would rerun and not change anything to the ccm data).

wrt to defined groups, when you are not using `ncm-accounts`, you could simply inject the data like the unittest does (maybe with a `kept_groups` like 
```pan
"/software/components/accounts/kept_groups/theadmins" = dict();
```
)
we could ofcourse extend the `defined_groups` to allow any string if `ncm-accounts` is not in use (at all or not active).
but then we should move the `defined_` functions out of the `ncm-accounts` functions imho.

i actually need to think about this a bit more, i'd like to play around with it a bit more. maybe we can merge this as tech preview :smile: 
(it reminds me that some changes in aii are required as well, PR soonish).